### ISSUE TITLE
[Bug Report] VOverflowBtn: Problematic style with prepend button

### ISSUE THREAD

**Issue by Tzahile** [vuetifyjs/vuetify#9448]
### Environment
**Vuetify Version:** 2.1.6
**Vue Version:** 2.6.10
**Browsers:** Chrome 77.0.3865.120
**OS:** Windows 10

### Steps to reproduce
Create a segmented overflow-btn (simply take [this](https://vuetifyjs.com/en/components/buttons#button-dropdown-variants) example by vuetify into codepen).
Add a prepend-icon prop to it

### Expected Behavior
The bottom line of the button, just like top line, will be all over the button

### Actual Behavior
Bottom line starts only where the button itself is, leaving the icon out

### Reproduction Link
<a href="https://codepen.io/Tzahile/pen/XWWRJop?editors=1011" target="_blank">https://codepen.io/Tzahile/pen/XWWRJop?editors=1011</a>

### Other comments
A workaround will be to give margin-left to .v-input__slot:before and change the width appropriately

**Comment by lioneaglesolutions** [comment]
### Stale Issue

This issue is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [[Discord community](https://community.vuetifyjs.com/)](https://community.vuetifyjs.com/).
